libs/vkd3d/command.c

```diff
@@ -6430,6 +6470,7 @@ static void STDMETHODCALLTYPE d3d12_command_list_CopyTiles(d3d12_command_list_if
             buffer, buffer_offset, flags);
 
     d3d12_command_list_end_current_render_pass(list, true);
+    d3d12_command_list_end_batch(list);
```

Comment:
I think this is missing ResolveSubresource as well?

Reply:
Added to resolve_subresource.